desktop/ui/src/main/java/org/datacleaner/windows/ResultWindow.java

```diff
@@ -359,15 +398,18 @@ protected JComponent getWindowContent() {
 
         final DCBannerPanel banner = new DCBannerPanel(imageManager.getImage("images/window/banner-results.png"),
                 bannerTitle);
-        _tabbedPane.bindTabTitleToBanner(banner);
+        _tabbedPane.addListener(new VerticalTabbedPane.Listener() {
+            @Override
+            public void stateChanged(int newIndex, Tab<?> newTab) {
+                banner.setTitle2(newTab.getTitle());
+                banner.updateUI();
+            }
+        });
 
         for (JComponent pluggableButton : _pluggableButtons) {
             banner.add(pluggableButton);
```

Comment:
Here the pluggable buttons are added to the banner. You didn't touch those, but maybe you should? :)
At least the only pluggable button we right now (in commercial editions) have is the "export metrics to csv" button (I think it's called that - or something very similar).... I think that fits in best with the other buttons that you moved.